File: fib.go
//usr/bin/go run $0 $@ ; exit

package main

import (
	"fmt"
)

func fib(n int) int {
	if n <= 1 {
		return n
	} else {
		return fib(n-1) + fib(n-2)
	}
}

func main() {
	fmt.Printf("result: %d\n", fib(40))
}

File: fib.c
#include <stdio.h>

int fib(int n) {
    if (n <= 1) {
        return n;
    } else {
        return fib(n-1) + fib(n-2);
    }
}

int main () {
    printf ("result: %d", fib (40));
}